test-series/1/test-1.1.0.ipynb

```diff
@@ -0,0 +1,1263 @@
+{
```

Comment:
<p>(see earlier comment about showing the full range of the FG line)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera-validation/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
<p>(see above comment).</p>